schemas/model_definition.schema.json

```diff
@@ -113,7 +115,9 @@
         "feature_names": {
           "type": "array",
           "items": {
-            "type": "string"
+            "type": "string",
+            "minItems": 0,
```

Comment:
Shouldn't `minItems` and `uniqueItems` be one level up ie. properties of `feature_names` not `items`?

https://json-schema.org/understanding-json-schema/reference/array.html#length